### ISSUE TITLE
Open tabs should be cleared when pervane is restarted with a different directory.

### ISSUE THREAD

**Issue by metanet** [hakanu/pervane#112]
Start pervane with `dir1` and open some tabs in the browser. Then, restart pervane with `dir2` and refresh the browser. This refresh does not clear the open tab list and if you click on any tab, it does not get the content. I think when pervane changes the working dir and it is refreshed on the browser, open tabs list should be cleared.

**Comment by hakanu** [comment]
Good catch. However, I guess this introduces some security implications. eg. we will start leaking user's "actual" dir passed as an arg into the javascript part.

So inclined not to do this. Any other ideas?

How i was originally planning:
1. Send root dir from server to js
1. In js if  root dir is different from previous one in the browser's localstorage.
  1.  delete all open tabs.

**Comment by metanet** [comment]
I dont think we need to expose root dir for that. We can create a random string for each root dir and put it into a file, for instance, into root_dir/.id and then send this random id to clients. We also dont show this file in the tree. With this approach I think you cannot expose the root dir from a random string.

Wdyt
